packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Convert to JS
```

Comment:
My response didn't end up on this thread:

> Yeah, but it was just easier to take the CS-built files and clean up a bit.
> I can later convert to TS from this rather than directly from CS.